Exercise 1 BFile/myFile-rt/src/com/tridiumuniversity/myFile/BFile.java

```diff
@@ -0,0 +1,216 @@
+package com.tridiumuniversity.myFile;
+
+import javax.baja.nre.annotations.*;
+import javax.baja.sys.*;
+
+@NiagaraType
+@NiagaraProperty(
+        name = "fileName",

```

Comment:
These properties look good, but it looks like the indentation is a bit inconsistent. The property attributes are indented 8 spaces, while the remainder of the code looks like it's using 2 spaces.